CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Features
 
 - Correctly render tab stops in --show-all, see #2038 (@Synthetica9)
+- Allow crates using `bat` as a library to configure `show_nonprintable` with `PrettyPrinter`, see #2142
```

Comment:
Move it to the `## `bat` as a library` section please :)

Reply:
No no, all good. I did have a glance at the sections but must have missed it! :sweat_smile: